### PR TITLE
Removal of WET v3.1 notice - close #6802

### LINKED ISSUE #6802: Remove 'Looking for WET v3.1' bar
Can we remove the 'Looking for WET 3.1' bar on every page and may be move that notice into a section in the [WET home page](http://wet-boew.github.io/wet-boew/index-en.html)

The issue is we got feedback from external user/developer at Guelph where they thought the WET was no longer an active project.

Cheers

:smile:

### PR DESCRIPTION
**Pull request by duboisp**
The notice was move into the documentation main page and a link to it was added in the home page under key resources..

The same text was intentionally kept as is, but the title was renamed to have a parallel structure for the key resources list.

Cheers

### CODE REVIEW

**Comment by LaurentGoderre** [comment]
Please revert the changes to the i18n.csv because they will get overwrittenh anyways next time a string changes. These strings are managed by a translation spreadsheet

**Comment by duboisp** [comment]
I reverted the changes I made in the i18n.cvs

How should/could change the translation spreadsheet? 

The following items are useless:
* tmpl-v31-ref-head
* tmpl-v31-ref-1
* tmpl-v31-ref-3

**Comment by LaurentGoderre** [comment]
We can leave it for now. The spreadsheet needs a big cleanup anyways

**Comment by duboisp** [comment]
@LaurentGoderre should I update my pull request with the change to i18n.csv?

**Comment by LaurentGoderre** [comment]
No, I don't think so. @pjackson28 are you ok with this change?

File: site/pages/docs/index-fr.hbs [hunk 1]
```diff
@@ -39,3 +39,8 @@ de l'information comment télécharger, utiliser ou contribuer à la boîte d'ou
 	<li>Suivre le <a href="ref/docstmpl-fr.html">modèle de documentation</a>.</li>
 	<li>Fournir au moins un exemple pour chaque attribut, thème ou élément de la BOEW. D’autres exemples doivent être donnés pour chaque paramètre ou option. </li>
 </ul>
+
+<section>
+	<h2 id="boew31">Avis concernant la BOEW v3.1</h2>
+	<p>À partir du 23 septembre 2014, la version 3.1 de la Boîte à outils de l'expérience Web n'est plus supportée. Le code source et la documentation ont été déplacés vers le dépôt <a href="{{site.legacyrepolink}}">{{site.legacyreponame}}</a>.</p>
+</section>
```
Comment (pjackson28) [inline site/pages/docs/index-fr.hbs]: Add a blank line to the end of the file (red icon above indicates it is missing).

File: site/pages/docs/index-en.hbs [hunk 2]
```diff
@@ -38,3 +38,8 @@
 	<li>Follow the <a href="ref/docstmpl-en.html">documentation example template</a>.</li>
 	<li>At least one working example should be provided for each WET feature, theme or component. Additional examples should be provided for each parameter or option.</li>
 </ul>
+
+<section>
+	<h2 id="wet31">Notice about WET v3.1</h2>
+	<p>As of September 23, 2014, version 3.1 of the Web Experience Toolkit is no longer supported. The source code and documentation have been moved to the <a href="{{site.legacyrepolink}}">{{site.legacyreponame}}</a>repository.</p>
+</section>
```
Comment (pjackson28) [inline site/pages/docs/index-en.hbs]: Add a blank line to the end of the file (red icon above indicates it is missing).

Comment (LaurentGoderre) [inline site/pages/docs/index-en.hbs]: You should also be using a text editor that support .editorconfig to make sure your code is formatted the right way

**Comment by pjackson28** [comment]
Yes we can go ahead with the change once the blank lines are added.

File: site/pages/docs/index-en.hbs (hunk 2 above)
Comment (duboisp) [inline site/pages/docs/index-en.hbs]: End of file fixed
Thanks @LaurentGoderre I added that plugin to my text editor

**Pull request merged**